--- vGizmo.h	2022-06-13 20:50:07.000000000 +0900
+++ /tmp/vGizmo.h	2022-06-13 20:51:35.000000000 +0900
@@ -85,7 +85,7 @@
         if(delta.x == 0 && delta.y == 0) update();
         delta.x = delta.y = 0;
     }
-    void inline testRotModifier(int x, int y, vgModifiers mod) { }
+    void inline testRotModifier(int x, int y, vgModifiers mod) { (void)x; (void)y; (void)mod; }
     
     //    Call on mouse button event
     //      button:  your mouse button
@@ -373,6 +373,7 @@
     //    Call on wheel (only for Dolly/Zoom)
     ////////////////////////////////////////////////////////////////////////////
     void wheel( T x, T y) {
+        (void)x;
         dolly.z += y * dollyScale * T(5);
     }
 
